### PR TITLE
Simple fix miss property...(class member)

### PR DESCRIPTION
**Pull request by xjaja**

### CODE REVIEW

**Comment by nikic** [comment]
This looks like spam. Please explain what you're trying to do if it isn't.

File: grammar/php7.y [hunk 1]
```diff
@@ -708,7 +708,7 @@ static_var:
 ;
 
 class_statement_list_ex:
-      class_statement_list_ex class_statement               { if ($2 !== null) { push($1, $2); } }
+      class_statement_list_ex class_statement               { if ($2 !== null) { push($1, $2); }; $$ = $1; }
```
Comment (nikic) [inline grammar/php7.y:711]:
```suggestion
      class_statement_list_ex class_statement               { if ($2 !== null) { push($1, $2); } else { $$ = $1; } }
```

**Review by nikic — COMMENTED** [review]
Okay, I understand what you are trying to do now. You need to run `grammar/rebuildParser.php` to regenerate the parser, otherwise this will have no effect.

**Comment by nikic** [comment]
This still needs to run grammar/rebuildParser.php to regenerate the parser.

**Comment by xjaja** [comment]
Thx.

**Pull request merged**